templates/about/manifesto.html

```diff
@@ -8,116 +8,123 @@
 
 {% block about_content %}
   <h1>The Open Operator Manifesto</h1>
-  <h2 class="p-heading--4 u-sv3" style="max-width: 40rem;">
-    <strong>We lead the Open Operator Collection, on a mission to elevate the art of dev&ndash;sec&ndash;ops with open&ndash;source operators that control enterprise applications.</strong> These values and guidelines ensure high&ndash;quality code, conversations and experiences for our community.
-  </h2>
+  <h2 class="p-heading--4 u-sv3" style="max-width: 40rem;">Better operators through open community standards</h2>
+  <p>Our goal is to improve devsecops with open source operators for widely used software. These shared values ensure high quality code and conversations.</p>
   <ol class="p-list--ordered is-h4">
     <li>
       Source required
-      <p>An operator is software that takes control of an enterprise system, application or service. The operator source code is essential to verify expected behavior. We make operator source available so that everybody can understand exactly what happens on their systems.</p>
+      <p>An operator is software that takes control of an enterprise system, application or service. The operator source code is essential to verify expected behavior.</p>
```

Comment:
@steverydz I think this one doesn't match the copy docs

Reply:
Ops, you removed the wrong one, hehe

Reply:
Fixed